app/controllers/passkit/api/v1/application_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Passkit
+  module Api
+    module V1
+      class ApplicationController < ActionController::API
+        after_action :allow_iframe
+
+        # The X-Frame-Options - header that indicate whether or not a browser should be allowed to render a page in a <frame>
+        def allow_iframe
+          response.headers.except! 'X-Frame-Options'
```

Comment:
Do you want to set this explicitly to not rely on default values that might change?
```suggestion
          response.headers['X-Frame-Options'] = 'SAMEORIGIN'
```

Reply:
The main reason of this PR is the problem with `SAMEORIGIN`, we need to delete it from headers as this header does not have suitable option, it has only `DENY` and `SAMEORIGIN`.